ister_gui.py

```diff
@@ -1110,6 +1114,11 @@ def update_widgets(self):
         if not dns_res:
             dns_res = 'none found'
 
+        if 'none found' in ''.join([interface_res, mask_res, ip_res, dns_res]):
```

Comment:
Hrm alrighty, I guess the 'Detected network configuration' is what threw me when it keyed off of 'none found' as they seemed to be opposing messages.